docs-src/index.md

```diff
@@ -1,36 +1,52 @@
 # Introduction
 
-Service APIs is the evolution of Kubernetes APIs that relate to `Services`,
-e.g. `Ingress`. The project is part of Kubernetes, working under
-[SIG-NETWORK][sig-network]. Use the following resources to learn more about
-Service APIs.
+Service APIs is an open source project, working under
+[SIG-NETWORK][sig-network], that is evolving service networking APIs within the Kubernetes ecosystem. Service APIs are the APIs you use to expose your Kubernetes applications - Services, Ingress, and more. The Gateway API is the primary API being developed to unify service networking under a core set of networking APIs.
```

Comment:
Naming is a bit confusing here. I think Service APIs covers GatewayClass, Gateway, *Route, and BackendPolicy resources. This represents an evolution of the Ingress API and takes on part of the functionality from the Service API, especially where `type=LoadBalancer`. I think it might be simplest to remove the last couple sentences in this paragraph, but maybe there's a better option.
```suggestion
[SIG-NETWORK][sig-network], that is evolving service networking APIs within the Kubernetes ecosystem.
```